asllib/error.ml

```diff
@@ -486,6 +492,13 @@ module PPrint = struct
     | NoLoopLimit ->
         fprintf f "@[%a@]" pp_print_text
           "ASL Warning: Loop does not have a limit."
+    | ConstraintSetPairToBigToBeExploded { op; left; right } ->
+        fprintf f
+          "@[Exploding@ sets@ in@ for@ the@ binary@ operation %s@ would@ \
+           result@ in@ too@ big@ a@ constraint@ set@ with@ constraints@ %a@ \
+           and@ %a. Continuing with those.@]"
```

Comment:
nit: grammar "in for", and perhaps "those" could more clearly state the incoming constraint sets will be used without exploding